src/EnergyPlus/HeatRecovery.cc

```diff
@@ -1551,6 +1551,8 @@ namespace HeatRecovery {
                                 } else {
                                     // need call to EMS to check node
                                     CheckIfNodeSetPointManagedByEMS(ExchCond(ExchNum).SecOutletNode, iHumidityRatioMaxSetPoint, LocalWarningError);
+                                    // TODO: What should be do here?
```

Comment:
Same here.

Reply:
As the warning message implies, without a setpoint on the node, the simulation continues without control of the desiccant heat exchanged to a maximum humidity ratio setpoint. So I assume it's fine to **not** recheck and throw at the end, but just want to make sure.